trade-finance-manager-api/src/v1/helpers/notify-template-formatters.js

```diff
@@ -14,37 +14,51 @@ const generateFacilitiesListString = (facilities) => facilities.reduce((acc, fac
 }, '');
 
 const generateBSSListString = (facilities) => {
-  const bssList = facilities.reduce((acc, facility) => {
-    const {
-      bondType, ukefFacilityID, bankReferenceNumber, uniqueIdentificationNumber,
-    } = facility;
-
-    const bankReference = uniqueIdentificationNumber || bankReferenceNumber;
-    const bankRefString = bankReference
-      ? `*Your bank ref: ${bankReference}\n`
-      : '';
-
-    return `${acc}*${bondType}\n${bankRefString}*UKEF facility ID: ${ukefFacilityID} \n\n`;
-  }, '');
-  const bssHeading = bssList ? '#Bond Support Scheme\n\n' : '';
-  return `${bssHeading}${bssList}`;
+  if (facilities.length) {
```

Comment:
I'm not sure why we need a facilities.length check. If its an empty array then the facilties.reduce would return ''